ledger-core/rms/proto_vnvn_validation_vcachedmemoryresponse.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package rms
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
+)
+
+func (m *VCachedMemoryResponse) Validate(currentPulse PulseNumber) error {
+	object := m.Object.GetValue()
+	_, err := validSelfScopedGlobalWithPulseBeforeOrEq(object, currentPulse, "Object")
+	if err != nil {
+		return err
+	}
+
+	switch m.CallStatus {
+	case CachedMemoryStateFound:
+		if m.Memory.IsEmpty() {
```

Comment:
`Class` isn't empty, too

Reply:
m.Class doens't exist at master

Reply:
and object is connected to the class, but I don't know how to check it

Reply:
=( add todo, please

Reply:
done